target/linux/ramips/dts/mt7620a_tplink_re200-v1.dts

```diff
@@ -0,0 +1,160 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "mt7620a.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "tplink,re200-v1", "ralink,mt7620a-soc";
+	model = "TP-Link RE200 v1";
+
+	aliases {
+		led-boot = &led_power;
+		led-failsafe = &led_power;
+		led-running = &led_power;
+		led-upgrade = &led_power;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,57600n8";
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_power: power {
+			label = "re200-v1:green:power";
+			gpios = <&gpio1 0 GPIO_ACTIVE_LOW>;
+		};
+
+		lan {
+			label = "re200-v1:green:lan";
+			gpios = <&gpio2 0 GPIO_ACTIVE_LOW>;
+		};
+
+		wlan {
+			label = "re200-v1:green:wlan";
+			gpios = <&gpio1 3 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "phy0radio";
+		};
+
+		qss {
+			label = "re200-v1:green:qss";
+			gpios = <&gpio1 15 GPIO_ACTIVE_LOW>;
+		};
+
+		wlan2g_red {
+			label = "re200-v1:red:wlan2g";
+			gpios = <&gpio0 2 GPIO_ACTIVE_LOW>;
+		};
+
+		wlan2g_green {
+			label = "re200-v1:green:wlan2g";
+			gpios = <&gpio3 0 GPIO_ACTIVE_LOW>;
```

Comment:
This is where I would put the trigger for 2g Wifi. Did you check whether phy0tpt works, as this would be preferred over phy0radio, which shows no activity (blinking).

Reply:
No, I didn't check it. IMHO phy0tpt makes sense for the general LED (as mentioned above) to indicate WiFi active or not and the radio triggers would make sense on the green 2G/5G LEDs - but the same problem with the 5G WiFi applies.

Reply:
Still, if there is a 2g led, I would use that with the appropriate trigger (phy0tpt or phy1tpt) if it is available on your device.

BTW, which is which, 2G and 5G vs. phy0 and phy1?

Regarding 5G, have you checked this approach:
https://github.com/openwrt/openwrt/blob/38bdfcdb87e9b3a1232e9820bd6534da4c28282b/target/linux/ramips/dts/mt7620a_iptime.dtsi#L74-L77 ?

Maybe @mans0n can help here, too.

Reply:
Try `led-sources = <2>;`. If the LED turns on even when wlan is off, then add `led-active-low;`
I haven't seen a device with two led per phy, so I'm not sure how to add both red/green leds.

Reply:
Didn't work, as mentioned below, I gave up on this.

Reply:
So, you really tried this:

```
&pcie0 {
	mt76@0,0 {
		reg = <0x0000 0 0 0 0>;
		mediatek,mtd-eeprom = <&radio 0x8000>;
		mtd-mac-address = <&uboot 0x1fc00>;
		mtd-mac-address-increment = <2>;
		ieee80211-freq-limit = <5000000 6000000>;

		led {
			led-sources = <2>;
			led-active-low;
		};
	};
};
```

?

Reply:
Yes, didn't work.
I flashed back to stock: the 5G LEDs don't work either. Maybe my device is bad?